packages/edit-site/src/components/editor/index.js

```diff
@@ -234,7 +234,7 @@ export default function Editor( { isLoading } ) {
 					<SiteEditorMoreMenu />
 					<EditorInterface
 						className={ clsx(
-							'edit-site-editor__interface-skeleton',
```

Comment:
Looks like WooCommerce uses this class?
https://wpdirectory.net/search/01HZF50T58QDM6WW7MP6WHQ7Y1

Reply:
They are actually applying the class them selves, so it should be fine 

https://github.com/woocommerce/woocommerce/blob/f7ef2ca91b79d17fe3ac821723a9b97e4cc680b5/plugins/woocommerce-admin/client/customize-store/assembler-hub/editor.tsx#L78

They probably don't need to apply it at all though, it seems useless looking at their code base.